tendrl/gluster_integration/manager/manager.py

```diff
@@ -200,6 +266,27 @@ def on_pull(self, raw_data, cluster_id):
                     index += 1
                 except KeyError:
                     break
+            # poplate the volume options
+            reg_ex = re.compile("^volume[0-9]+.options+")
```

Comment:
I would really like to see this implemented as part of the main loop abd get this updated along with update_volume. The advantage we get is that we save 'n' number of extra updates to etcd. Is it something doable or you see any issues here?